Source/init.cpp

```diff
@@ -90,7 +90,7 @@ std::vector<std::string> GetMPQSearchPaths()
 	if (paths[0] == paths[1])
 		paths.pop_back();
 
-#if defined(__linux__) && !defined(__ANDROID__)
+#if (defined(__linux__) && !defined(__ANDROID__)) || defined(__FreeBSD__) || defined(__DragonFly__) || defined(__NetBSD__) || defined(__OpenBSD__)

```

Comment:
Could this just use `__unix__ && !__ANDROID__`? Looks like all linux and most BSD systems define unix (macos doesn't but that's probably desirable anyway for this case?)

Reply:
Makes sense.

Reply:
Seams good mac has its own scheme for this